forms-flow-nav/src/sidenav/Sidebar.scss

```diff
@@ -219,16 +255,17 @@ $animSpeed: 300ms;
           flex-grow: 1;
           width: 2rem;
           
-          color: var(--navbar-main-menu-active-font-color) !important;
+          // color: var(--navbar-main-menu-active-font-color) !important;
+          color: #525254;
           font-weight: 400;
           font-size: $fontSmallest;
           line-height: $fontLineHeight;
```

Comment:
could plese use the mixin available in the _theme file.
and just pass size weith and color as css variable in mixin. 
like @include font-style(--font-size-xl, --font-weight-semibold, --red-100  );